packages/table/src/cell/_common.scss

```diff
@@ -29,7 +29,7 @@ $cell-transition-duration: $pt-transition-duration * 3;
   transition: color $cell-transition-duration;
 
   .pt-dark & {
-    &:not([class*="pt-intent-"]) {
+    &:not([class*="pt-intent-"]):not(.pt-loading) {
```

Comment:
that syntax is proposed in selectors level 4: https://drafts.csswg.org/selectors-4/#negation

in selectors level 3, `:not` only takes a simple argument. so while this compiles and seems to work in chrome and FF, i'm on the latest stable version for both. i'm going to go with the more verbose syntax for now since it's guaranteed it'll work on older browsers